lib/task_listener.js

```diff
@@ -181,6 +182,18 @@ export default class TaskListener extends EventEmitter {
               stack: e.stack
           });
         }
+        let stats = {
+          uptime: this.host.billingCycleUptime(),
+          interval: this.host.billingCycleInterval()
+        };
+        let remainder = stats.interval - (stats.uptime % stats.interval);
+        if(remainder * this.runtime.taskQueue.slowdownDivisor > stats.interval) {
```

Comment:
@gregarndt Any ideas on how to do this? The uptime and interval can't really be adjusted for testing purposes, so I'm not really sure how to test this without waiting for 40 minutes for the slowdown period to begin on its own.